tests/integration/nm/bond_test.py

```diff
@@ -78,9 +78,26 @@ def _bond_interface(name, options):
 
 
 @nmclient_context
-def _get_bond_current_state(name):
+def _get_bond_current_state(name, option=None):
+    """
+    When option defined, the returned state will only contains the
+    specified bond option and the bond mode.
+    When option not defined, the return state will only contains bond mode.
+    This is needed for assert check.
+    """
     nmdev = nm.device.get_device_by_name(name)
     nm_bond_info = nm.bond.get_bond_info(nmdev) if nmdev else {}
+    if not nm_bond_info:
+        return {}
+    bond_options = nm_bond_info[schema.Bond.OPTIONS_SUBTREE]
+    nm_bond_info[schema.Bond.OPTIONS_SUBTREE] = {
+        schema.Bond.MODE: bond_options[schema.Bond.MODE]
+    }
+    if option:
+        nm_bond_info[schema.Bond.OPTIONS_SUBTREE] = {
+            schema.Bond.MODE: bond_options[schema.Bond.MODE],
+            option: bond_options[option],
+        }
```

Comment:
This should be:
```
nm_bond_info[schema.Bond.OPTIONS_SUBTREE][option] = bond_options[option]
```

Reply:
why?

Reply:
Because before the condition you got:
```
    nm_bond_info[schema.Bond.OPTIONS_SUBTREE] = {
        schema.Bond.MODE: bond_options[schema.Bond.MODE]
    }
```
So in the condition body you just need to add the options and nothing more.